### ISSUE TITLE
Using graphics-smooth with pixi-legacy 6.5 with typescript won't build

### ISSUE THREAD

**Issue by branj** [pixijs/graphics-smooth#30]
Hey, thanks for this project. Just wanted to share a potential issue: 

I upgraded to pix.js-legacy@6.51 (to prep for 7 and offer canvas fallback support) and was getting errors out of the graphics-smooth lib when trying to build (this is a TS project): 

```
node_modules/@pixi/graphics-smooth/index.d.ts:225:22 - error TS2415: Class 'SmoothGraphics' incorrectly extends base class 'Container<DisplayObject>'.
  Property '_renderCanvas' is protected in type 'SmoothGraphics' but public in type 'Container<DisplayObject>'.

225 export declare class SmoothGraphics extends Container {
```

This is from `@pixi/graphics-smooth/index.d.ts`
![Screen Shot 2022-12-16 at 3 06 02 PM](https://user-images.githubusercontent.com/1408792/208203032-be2e4dcb-b7d0-4ce7-b6e5-bd53a663aeaa.png)

I experimented with many different versions to come to the conclusion I think it's an actual issue. Changing `_renderCanvas` to public, or simply removing any access modifiers fixed this issue.

**Comment by miltoncandelero** [comment]
I am not sure this can work with pixijs legacy... the whole point of this package is to use super cool shaders that don't exist on canvas 🤔

**Comment by ivanpopelyshev** [comment]
yeah, we should fix it, canvas is "legacy" in only one meaning - it was in core, now its separated. We still need it at least for the tests against Skia.
